recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ source:
     - 0103-BUILD-changes-for-external-sqlite-package.patch
     - 0104-Fixed-linking-error-in-conda-builds.patch
     - 0106-Upstream-patch-for-absl-error.patch            #[x86_64]
+    - 0007-do-not-hardcode-ca-cert-location.patch
```

Comment:
Is patch file name convention correct for this patch? I think it should be 01xx.

Reply:
yup! Thanks! Fixed.